### ISSUE TITLE
Error when attempting to update the details of a Site with no changes.

### ISSUE THREAD

**Issue by bdlamprecht** [digitalocean/pynetbox#63]
### Issue type
[X] Bug report      

### Environment
* Python version:  3.5
* NetBox version:  2.3.3
* PyNetBox version:  3.3.1

### Description
When attempting to update the details of an object within the model `dcim.sites` without any changes, I get the following HTML error:
```
int() argument must be a string, a bytes-like object or a number, not &#39;dict&#39;
```

If I manually change a single character in any of the fields using the `WebUI` of NetBox and re-run the same `pynetbox` code, everything is updated perfectly.

This is the actual cause of issue #60, not the "odd characters" that was previously mentioned.
I can perform the same action (sending data that is the same) using `curl` via a `PUT` without any issues.

However, whenever the `nb_site.save()` is called with no actual changes to the object, the above error is given.
This is especially difficult to handle, because the error isn't a `pynetbox.lib.query.RequestError` which I catch, but a generic server error.

I will close the other issue shortly once this gets assigned an issue number.

**Comment by zachmoody** [comment]
This issue should be fixed in #71. We'll treat #60 separately for now.

**Comment by bdlamprecht** [comment]
Yeah, that was it, all good now.
You rock @zachmoody !
